source/Octopus.Tentacle/Configuration/WindowsMachineKeyEncryptor.cs

```diff
@@ -7,10 +7,12 @@ namespace Octopus.Tentacle.Configuration
 {
     public class WindowsMachineKeyEncryptor : IMachineKeyEncryptor
     {
+#pragma warning disable CA1416
```

Comment:
This comes from a bump of some nuget packages in Halibut to add the Memory Cache